data/language/en-GB.txt

```diff
@@ -3642,6 +3642,9 @@ STR_6383    :Open download page
 STR_6384    :Snow
 STR_6385    :Heavy Snow
 STR_6386    :Blizzard
+STR_6387    :Can't lower element here...
```

Comment:
Should have used an ellipsis here. (`…`)

Reply:
To bem fair I copied from the "can't raise land here..." text, so we probably need to update it there